pkg/systemd/graphs.html

```diff
@@ -106,7 +106,7 @@
                         <nav class="pf-c-breadcrumb" aria-label="breadcrumb">
                             <ol class="pf-c-breadcrumb__list">
                                 <li class="pf-c-breadcrumb__item">
-                                    <a role="link" tabindex="0" translate='yes' class="pf-c-breadcrumb__link" id="system-link">Overview</a>
+                                    <a role="link" tabindex="0" translate='yes' class="pf-c-breadcrumb__link" id="overview-system-link">Overview</a>
```

Comment:
That's an a bit unhappy name, as both of them refer to the Overview -- just one comes from the graph summary, the other from an individual graph.